shell/platform/glfw/platform_handler.cc

```diff
@@ -73,6 +74,9 @@ void PlatformHandler::HandleMethodCall(
     }
     glfwSetClipboardString(window_, itr->value.GetString());
     result->Success();
+  } else if (method.compare(kSystemNavigatorPopMethod) == 0) {
+    glfwSetWindowShouldClose(window_, true);
```

Comment:
I believe we actually want to exit the process based on the description (that's what the macOS implementation does). In a simple runner like FDE's this will do that, but that's only because that runner exits as soon as the window closes.

I'm fine landing this as-is now and revisiting if necessary though, since in the common case it'll do the right thing.